src/seed-script.ts

```diff
@@ -0,0 +1,24 @@
+import { faker } from '@faker-js/faker';
+import { AppDataSource } from './data-source';
+import { User } from './entity/User';
+import { startDB } from './server';
+import * as dotenv from 'dotenv';
+
+async function seeder() {
```

Comment:
```suggestion
async function seedUsers() {
```

Sugestão tanto pra função ser um verbo como pra especificar melhor de qual seed se trata.